specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1199 @@
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/ClusterResponse"
+            }
+          },
+          "202": {
+            "description": "accepted operation"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Clusters_CreateOrUpdate": {
+            "$ref": "./examples/Clusters_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Clusters_Update",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "Clusters"
+        ],
+        "summary": "Update a cluster in a private cloud",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/clusterName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/ClusterResponse"
+            }
+          },
+          "202": {
+            "description": "accepted operation"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Clusters_Update": {
+            "$ref": "./examples/Clusters_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Clusters_Delete",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "Clusters"
+        ],
+        "summary": "Delete a cluster in a private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/clusterName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
```

Comment:
200 ok response missing. 204 indicates request was well formed but there was nothing to delete. 200 indicates that the resource was successfully deleted.

Reply:
I added this to the `2019-08-09-preview` API.